app/layouts/DashboardLayout/BalanceIndicator.view/BalanceIndicator.test.tsx

```diff
@@ -1,20 +1,19 @@
-/*
-
 import React from 'react';
 
 import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import '../../../testUtils/i18nForTests';
 
-import BalanceIndicator from '../../../../app/views/wallet/DashboardView/BalanceIndicator';
+import { BalanceIndicator } from './BalanceIndicator.view';
 
-const MOCK_BALANCE = '1234230000000';
+const MOCK_BALANCE = '10.000000000000';
 
 describe('BalanceIndicator', () => {
   test('renders the correct balance', () => {
     render(<BalanceIndicator balance={MOCK_BALANCE} isSynced />);
```

Comment:
I think we also need a test showing the "unsynced" message if `isSynced===false`

Reply:
Thats on line 25.